docs/transition/transitions.stories.ts

```diff
@@ -0,0 +1,115 @@
+import {
+  ref,
+  provide,
+  inject,
+} from 'vue';
+import { actions } from '@storybook/addon-actions';
+import UiText from '@/components/atoms/UiText/UiText.vue';
+import UiButton from '@/components/atoms/UiButton/UiButton.vue';
+import './transitions.stories.scss';
+
+const events = actions({ onUpdateModelValue: 'update:modelValue' });
+
+export default {
+  title: 'Utilities/Transitions',
+  args: { initModelValue: false },
+  decorators: [ (story, { args }) => ({
+    components: {
+      story,
+      UiButton,
+    },
+    setup() {
+      const modelValue = ref(args.initModelValue);
+      const toggleHandler = () => {
+        modelValue.value = !modelValue.value;
+      };
+      provide('modelValue', modelValue);
+      return { toggleHandler };
+    },
+    template: `<div class="min-h-80">
```

Comment:
IMO `min-h-80` is too high. Maybe half of it should be ok? Because now we have a lot of empty space 🙈

Reply:
Sure, I'm changing